src/requires-app-auth.ts

```diff
@@ -1,6 +1,7 @@
 const PATHS = [
   "/app",
   "/app/hook/config",
+  "/app/hook/deliveries",
```

Comment:
I decided it's not work adding all the extra logic just set JWT auth on custom paths. We might need it in future, but it's too much work and to low of a priority to work on it right now. Let's just add these paths and be done with it 😆 

Can you add the other two paths please?

```suggestion
  "/app/hook/deliveries",
  "/app/hook/deliveries/{delivery_id}",
  "/app/hook/deliveries/{delivery_id}/attempts",
```